common/web/types/src/package/kps-file.ts

```diff
@@ -184,7 +184,17 @@ export interface KpsFileLanguageExample {
      */
     ID: string;
     /**
-     * A space-separated list of keys, modifiers indicated with "+", spacebar is "space", plus key is "shift+=" or "plus"
+     * A space-separated list of keys.
```

Comment:
Is this now an array of KeyboardInfoFileExampleKey?

Reply:
No, this is as the comment describes. It gets converted by kmc-keyboard-info to KeyboardInfoFileExampleKey when building the .keyboard_info file.